panel/widgets/chat.py

```diff
@@ -720,12 +724,12 @@ def _build_entry(
             value: Union[ChatEntry, dict, Any],
             user: Optional[str] = None,
             avatar: Optional[Union[str, BinaryIO]] = None,
-        ) -> ChatEntry:
+        ) -> Optional[ChatEntry]:
```

Comment:
```suggestion
        ) -> ChatEntry | None:
```

I would call `Optional` soft deprecated, as this style is preferred.

Reply:
Agree. It was just the One used already

Reply:
We should go through and update this consistently everywhere. Here's all the changes I'd like to make:

- Remove all usage of `Optional`
- Remove all usage of `Union`
- Always use actual types rather than referring to the type as a string

Reply:
It should be easy to do with Ruff: `ruff --select=UP --fix --target-version=py38 panel`. I chose all of the settings as this will also convert to f-strings if possible. For a more specific selection, see [here](https://beta.ruff.rs/docs/rules/#pyupgrade-up).

Always having a future import with isort at the start of the file, should also be done.

Reply:
Fixed in the chat.py file. Changing it anywhere else is out of scope for this PR.